bugbug/tools/code_review.py

```diff
@@ -125,10 +125,12 @@ def __init__(self, patch_id) -> None:
 
 class Patch:
     raw_diff: str
+    file_diff: str
```

Comment:
Why do we need `file_diff` here? The `raw_diff` should have the full diff in git format.

Reply:
removed it